src/main/java/uk/gov/hmcts/reform/cpo/repository/CasePaymentOrdersRepositoryImpl.java

```diff
@@ -28,6 +28,8 @@ public CasePaymentOrdersRepositoryImpl(CasePaymentOrdersJpaRepository casePaymen
     public void deleteByUuids(List<UUID> uuids) {
         int deleteByIds = casePaymentOrdersJpaRepository.deleteByIdIsIn(uuids);
 
+        casePaymentOrdersJpaRepository.flush();
```

Comment:
may be you can remove flush call now?

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/63/commits/63b750b2aef7a52fd98521c1365f4a7f3c8af243